api/src/main/resources/application.yml

```diff
@@ -5,5 +5,9 @@ server:
 mongodb:
   database: "mydb"
   collection: "message"
+  uri: "mongodb://34.78.196.126:27017"
```

Comment:
So, instances from app engine can contact compute engine without any restriction?

Reply:
I don't understand what is restriction in this case, but there is exist another way to connect both engine - using internal IP and VPC connector